sklearn/model_selection/_validation.py

```diff
@@ -1500,6 +1500,32 @@ def learning_curve(
     -----
     See :ref:`examples/model_selection/plot_learning_curve.py
     <sphx_glr_auto_examples_model_selection_plot_learning_curve.py>`
```

Comment:
I just wonder if referring to the example here is really necessary. If you feel like doing so, maybe you can take the opportunity to erase it.